recipe/meta.yaml

```diff
@@ -1,46 +1,57 @@
 {% set name = "sparse" %}
-{% set version = "0.13.0" %}
-{% set sha256 = "c1f37c2097dfb017789ade963cebd199151b015aa6a30eece2359eb4e3c545e1" %}
+{% set version = "0.15.1" %}
+{% set sha256 = "973adcb88a8db8e3d8047953331e26d3f64a5657f9b46a6b859c47663c3eef99" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/pydata/{{ name }}/archive/{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  # skip as per upstream req, also missing numba >=0.49 for s390x
+  skip: True  # [py<38 or s390x]
 
 requirements:
-  build:
-    - python >=3.7
+  host:
+    - python
     - pip
+    - wheel
+    - setuptools >=64
+    - setuptools-scm >=8
   run:
-    - python >=3.7
+    - python
     - numpy >=1.17
     - scipy >=0.19
     - numba >=0.49
 
 test:
+  source_files:
+    - sparse/tests
   imports:
     - sparse
+  commands:
+    - pip check
+    - pytest sparse/tests
```

Comment:
I think that if you were to run `pytest -v --pyargs sparse`, you woulnd't have to include the tests with `source_files` (since they are part of the parse python package directly) and pytest will be able to import the tests.

Reply:
I pushed a commit to do just that.